hoodie-client/src/main/java/com/uber/hoodie/io/compact/HoodieRealtimeTableCompactor.java

```diff
@@ -145,14 +163,23 @@
 
     TableFileSystemView.RealtimeView fileSystemView = hoodieTable.getRTFileSystemView();
     log.info("Compaction looking for files to compact in " + partitionPaths + " partitions");
-    List<CompactionOperation> operations = jsc.parallelize(partitionPaths, partitionPaths.size())
-        .flatMap((FlatMapFunction<String, CompactionOperation>) partitionPath -> fileSystemView
-            .getLatestFileSlices(partitionPath).map(
-                s -> new CompactionOperation(s.getDataFile().get(), partitionPath,
-                    s.getLogFiles().sorted(HoodieLogFile.getLogVersionComparator().reversed())
-                        .collect(Collectors.toList()), config))
-            .filter(c -> !c.getDeltaFilePaths().isEmpty()).collect(toList()).iterator()).collect();
+    List<CompactionOperation> operations =
```

Comment:
identation change?

Reply:
Since the change of code is inside the closure, I think the whole code piece is considered changed..

Reply:
I see. gtk